README.md

```diff
@@ -71,6 +71,20 @@ The exported exceptions need to be exposed via an HTTP endpoint in order for Per
 See [Periskop Documentation](https://github.com/soundcloud/periskop) for more information on how to configure a
 Periskop server.
 
+### Using push gateway
+
+You can also use [pushgateway](https://github.com/soundcloud/periskop-pushgateway) in case you want to push your metrics 
+instead of using pull method. Use only in case you really need it (e.g a batch job) as it would degrade the performance 
```

Comment:
would -> could? (I guess it does not have to if you flush events in the background?) but in any case, pull is the recommended approach since it is more scalable